main.go

```diff
@@ -45,39 +42,54 @@ func main() {
 	}
 	cs := kubernetes.NewForConfigOrDie(config)
 
-	watch, err := cs.CoreV1().Pods("").Watch(context.Background(), metav1.ListOptions{})
+	watch, err := cs.CoreV1().Pods("").Watch(ctx, metav1.ListOptions{})
 
 watchloop:
 	for evt := range watch.ResultChan() {
 		pod := evt.Object.(*v1.Pod)
-		switch evt.Type {
-		case "ADDED", "MODIFIED":
-			for _, status := range pod.Status.ContainerStatuses {
-				if w := status.State.Waiting; w != nil {
-					if w.Reason == "CrashLoopBackOff" {
-						log.Printf("Pod %v/%v is in CrashLoopBackOff, deleting", pod.Namespace, pod.Name)
-						for {
-							if _, _, err = chat.PostMessage(*channelId, slack.MsgOptionText(
-								fmt.Sprintf("Deleting crashlooping pod `%s/%s`", pod.Namespace, pod.Name), false)); err != nil {
-								var rlError *slack.RateLimitedError
-								if errors.As(err, &rlError) {
-									log.Printf("Rate limited, sleeping %s", rlError.RetryAfter)
-									time.Sleep(rlError.RetryAfter)
-									break
+		if v, found := pod.Labels["crashloopbackon"]; found {
```

Comment:
Wasn't there supposed to be a com.apollographql/?

Reply:
It felt unnecessary.